scripts/globals/spells/paralyze_ii.lua

```diff
@@ -57,18 +47,21 @@ function onSpellCast(caster,target,spell)
         params.effect = dsp.effects.PARALYSIS;
         local resist = applyResistanceEffect(caster, target, spell, params);
 
+    if (duration >= 90) then
```

Comment:
We're defining it as 120 then checking if its greater than 90? I see nothing that would reduce it from 120 before reaching this conditional

Reply:
please fix the indent while you are here, it really throws me off trying to read this

Reply:
changed indents but i followed the same rules allready applied for blind 2 and slow 2 for the duration part, So not sure if you want to scrap that part completely of not

Reply:
I dunno when those happened. Its wrong.

Reply:
```
     duration = duration * applyResistanceEffect(caster, target, spell, params);
 
     if (duration >= 90) then --Do it!
```
is what blind 2 does